README.md

```diff
@@ -63,7 +63,12 @@ rx2-channel:                   # Rx2 channel (optional)
   max-data-rate: 5
   radio: 0
 rx2-default-data-rate: 0       # Default data rate index of Rx2 (optional)
-max-eirp: 29.15                # Maximum EIRP to use regardless of band settings (optional)
+max-eirp: 29.15                # Maximum EIRP for this frequency plan (optional)
+sub-bands:                     # Sub-bands override the duty cycle and maximum EIRP for frequency ranges.
+- min-frequency: 868000000     # Minimum frequency (Hz)
+  max-frequency: 868600000     # Maximum frequency (Hz)
+  duty-cycle: 0.01             # Duty cycle for this sub-band (optional)
+  max-eirp: 16.15              # Maximum EIRP for this sub-band (optional)
```

Comment:
It's officially under `band-id` (so in the top)

Reply:
What do you mean?

Reply:
Comment on the wrong line, but I meant to move `sub-bands` to right under `band-id`